spec.bs

```diff
@@ -384,7 +384,8 @@ const request = new PaymentRequest([{
     challenge: new Uint8Array([21,31,105 /* 29 more random bytes generated by the bank */]),
 
     instrument: {
-      displayName: "Fancy Card ****1234",
+      displayName: "FancyBank Platinum Card",
+      details: "****1234 | 01/29",
```

Comment:
Open question: is `details` a good name here? Should we call it `displayDetails`?

Reply:
I have no strong feeling about the name. I note below that you wrote "Optional additional detail string" so another name idea is 'additionalDetails'.

Reply:
Ack, let's just stick with `details` then.